### PR TITLE
Add Strata integration examples

### PR DESCRIPTION
**Pull request by ChewingGlass**
Will add more examples later, but this should get people started.

### CODE REVIEW

**Comment by 0xPratik** [comment]
Hello Ser, @ChewingGlass The link is Broken here [Strata Launchpad](https://app.strata.launchpad.com/launchpad)

File: docs/integrations/strata.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+Strata is a protocol for launching tokens built on Solana. 
+You can use Strata to launch any kind of fungible token, ranging from social tokens to dao and gamefi tokens.
+You can also compose strata with anything that uses fixed price mechanics to get dynamic pricing mechanics, for example the Metaplex CandyMachine.
+
+More in-depth docs are available [here](docs.strataprotocol.com). You can also use the gui at [Strata Launchpad](app.strataprotocol.com)
+
+## How to create a fully managed token
+
+A fully-managed Strata token is a token where the liquidity is managed by the protocol. The upshot is that you immediately get a tradeable
```
Comment (jacobcreech) [inline docs/integrations/strata.md:49]: So is a Strata token a wrapped spl-token? How does this work?

File: code/strata/fully-managed/create-token.en.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  // Price = 0.01 * sqrt(Supply)
+  const curve = await tokenBondingSdk.initializeCurve({
+    config: new ExponentialCurveConfig({
+      c: 0.01,
+      b: 0,
+      pow: 1,
+      frac: 2,
+    }),
+  });
+  const { tokenBonding } = await tokenBondingSdk.createTokenBonding({
```
Comment (jacobcreech) [inline code/strata/fully-managed/create-token.en.ts:18]: Could we get a log after here to help understand what we just created? Just add a comment of the log output

**Review by jacobcreech — COMMENTED** [review]
Few Nits. Mostly clarifying questions so I understand what is being added. Thanks for the contribution thus far!

**Comment by jacobcreech** [comment]
As a side note - The main vision for the integrations section of cookbook is to facilitate CPI examples for people use in their own programs. Would be awesome to have some basic CPI examples for interacting with Strata

**Comment by ChewingGlass** [comment]
I'll look to add those in the future. Right now we don't have many people doing CPIs, as most composability happens at the sdk layer. I.e. chaining a bonding curve into a candymachine to make a dynamically priced candymachine

**Review by jacobcreech — APPROVED** [review]
Thanks for the contribution!

**Pull request merged**